tests/ui/never_type/lint-never-type-fallback-flowing-into-unsafe.e2024.stderr

```diff
@@ -0,0 +1,116 @@
+error: never type fallback affects this call to an `unsafe` function
+  --> $DIR/lint-never-type-fallback-flowing-into-unsafe.rs:13:18
+   |
+LL |         unsafe { mem::zeroed() }
+   |                  ^^^^^^^^^^^^^
+   |
+   = warning: this will change its meaning in a future release!
```

Comment:
this... is a bit weird.

for e2024, the behavior _won't_ change, so this is misleading...